src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/summarize.py

```diff
@@ -159,8 +162,35 @@ def collect_statistic(root: ET.Element, is_conformance_mode: bool):
     return devices, results, general_pass_rate, pass_rate_avg, general_test_count, trusted_ops, covered_ops
 
 
-def create_summary(summary_root: ET.Element, output_folder: os.path, report_tag: str, is_conformance_mode: bool,
-                   output_filename='report'):
+def format_string(input_str: str):
+    res = input_str
+    res = res.replace('{', '')
+    res = res.replace('}', '')
+    res = res.replace("'", '')
+    res = res.replace('"', '')
+    res = res.replace(': ', '=')
+    res = res.replace(' ', '')
+    res = res.replace(',', ' ')
+    return res
```

Comment:
Could we pass between functions a dictionary of results in the form of {'Add': {'passed': 10, 'failed': 0}, ...}